lang/en/quizaccess_proctor.php

```diff
@@ -35,7 +35,8 @@
 $string['selectproctor'] = 'Select Proctoring Type';
 $string['noproctor'] = 'No Proctoring';
 $string['aiproctor'] = 'AI Proctoring';
```

Comment:
@rohansharmasitoula I don't think `aiproctor` string is being used anywhere else, please validate and remove